docs/user/limitations/limitations.rst

```diff
@@ -54,6 +54,17 @@ For example, e.g. `SELECT depo.name, avg(empo.age) FROM empo JOIN depo WHERE emp
 Here's a link to the Github issue - [Issue 110](https://github.com/opendistro-for-elasticsearch/sql/issues/110).
 
 
+Limitations on Window Functions
+===============================
+
+For now it's not allowed to use non-regular fields (calculated by scalar or aggregated function) in window definition. For example, ``avg_flight_time`` is not accessible to the rank window definition as follows::
```

Comment:
How about this? 
For now, only the field defined in index is allowd, all the other caculated fields (calculated by scalar or aggregated function) is not allowed.

Reply:
Sure, done!